### PR TITLE
Add option to not add VersioningIterator 

### PR DESCRIPTION
**Pull request by avgAGB**

### CODE REVIEW

File: warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java [hunk 1]
```diff
@@ -67,6 +67,14 @@ public static void setPropertyIfNecessary(String tableName, String propertyName,
         }
     }
 
+    public static void disableVersioningIfNecessary(TableOperations tops, Logger log) throws AccumuloException, AccumuloSecurityException {
+
+        for (IteratorScope iterScope : IteratorScope.values()) {
+            tops.removeProperty(Property.TABLE_ITERATOR_PREFIX + iterScope.name() + ".vers", "20," + VersioningIterator.class.getName());
```
Comment (jschmidt10) [inline warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java]: The API says that the first argument is the tableName and the 2nd is the property name. You don't need to specify the value of the property to remove.

File: warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java [hunk 2]
```diff
@@ -67,6 +67,14 @@ public static void setPropertyIfNecessary(String tableName, String propertyName,
         }
     }
 
+    public static void disableVersioningIfNecessary(TableOperations tops, Logger log) throws AccumuloException, AccumuloSecurityException {
```
Comment (jschmidt10) [inline warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java]: The log argument appears to be unused. I'd remove it if we don't need it.

File: warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java (hunk 1 above)
Comment (jschmidt10) [reply warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java]: It looks like Accumulo has a nicer API for this so you don't have to hard-code the options to remove. Check out `TableOperatoins.removeIterator()`.

File: warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java (hunk 2 above)
Comment (avgAGB) [reply warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java]: Thanks, will do

File: warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java (hunk 1 above)
Comment (avgAGB) [reply warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java]: Thanks for the heads up. I'll adjust this and test with the new call.

**Pull request closed without merging**